kinde_fastapi/examples/management_token_example.py

```
@@ -0,0 +1,119 @@
+        )
+    except Exception as e:
+        logger.error(f"Exception in token validation: {str(e)}", exc_info=True)
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            detail=f"Token introspection failed: {str(e)}",
+            headers={"WWW-Authenticate": "Bearer"}
+        )
+
+# Example route using the validated management token
+@app.get("/management/users")
+async def get_users(token_manager: ManagementTokenManager = Depends(get_management_token)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)